### PR TITLE
Adding My Project

### PR DESCRIPTION
**Pull request by niranjannitesh**
Preview URL: http://niteshkumarniranjan.pythonanywhere.com/

### CODE REVIEW

**Comment by niranjannitesh** [comment]
@poush @djmgit Please Review this.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
**Randomly, It gives me :**

image

> It's leaking, I think. Same set of `string` gives mixed responses.

**Comment by 0x48piraj** [comment]
**I made an `.gif`of the bug. Hope it helps!**

**Comment by niranjannitesh** [comment]
@0x48piraj Check it again.

File: niteshkumarniranjan/app.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+app = Flask(__name__)
+app.secret_key = "sdsafsfdsfbdsj!jsdkfl"
+
+class simple_form(FlaskForm):
+    name = StringField('Name', validators=[DataRequired()])
+
+
+@app.route("/", methods=('GET', 'POST'))
+def index():
+    form = simple_form()
```
Comment (vaibhavsingh97) [inline niteshkumarniranjan/app.py:19]: So If i submit form than, it should promt for user to enter text

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by niranjannitesh** [comment]
@vaibhavsingh97 Fixed That 
screenshot 96

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by 0x48piraj — APPROVED** [review]
Seems good now :+1: 

Thanks for your patience :heart:

**Review by vaibhavsingh97 — APPROVED** [review]

**Pull request merged**